test/e2e/apimachinery/webhook.go

```diff
@@ -1573,3 +1568,33 @@ func testSlowWebhookTimeoutNoError(f *framework.Framework) {
 	err = client.CoreV1().ConfigMaps(f.Namespace.Name).Delete(name, &metav1.DeleteOptions{})
 	gomega.Expect(err).To(gomega.BeNil())
 }
+
+// createAdmissionWebhookMultiVersionTestCRDWithV1Storage creates a new CRD specifically for the calling test.
+func createAdmissionWebhookMultiVersionTestCRDWithV1Storage(f *framework.Framework, pruning bool) (*crd.TestCrd, error) {
```

Comment:
pruning is unused... coming in the pruning PR?

Reply:
right. Doesn't hurt now.

Reply:
fixed